docs/guides/examples/quickstart/frontend/package.json

```
@@ -13,7 +13,7 @@
     "@types/node": "^16.18.8",
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
-    "fuels": "^0.79.0",
+    "fuels": "^0.82.0",
```

I think it should allow patches only, no?

```suggestion
    "fuels": "~0.82.0",
```